jenkins-client-it-docker/src/test/java/com/offbytwo/jenkins/integration/NoExecutorStartedGetJobDetailsIT.java

```diff
@@ -25,6 +25,10 @@ public void beforeMethod() throws IOException {
     @Test
     public void shouldCheckTheBuildCause() throws IOException {
         BuildWithDetails details = job.getFirstBuild().details();
+        assertThat(details.getArtifacts()).isNotNull();
+        assertThat(details.getArtifacts().size()).isEqualTo(2);
+        details.getArtifacts().forEach(a -> assertThat(a.getClient()).isNotNull());
+
```

Comment:
Is there a good reason why you changed the behaviour of this integration test?

Reply:
Yes, this is required to test that "artifact.getClient()" is properly set (i.e. not null). But I guess it might be better to have a dedicated test method instead?